static/js/people.js

```diff
@@ -723,28 +723,28 @@ exports.get_realm_users = function () {
     return Array.from(active_user_dict.values());
 };
 
-exports.get_active_humans = function () {
-    const humans = [];
```

Comment:
why don't you initialize this array in exports.initialize and just return here instead of filtering it from the dictionary?

Reply:
@joaomcarvalho No, the set of active users can change after initialization via `realm_user` events from the server.